[DOC] Improve the docs section about upgrading resources from v1alpha1 to v1beta1 (#3911)
strimzi/strimzi-kafka-operator: documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

[id='assembly-upgrade-resources-{context}']
= Strimzi resource upgrades

The `{KafkaApiVersionPrev}` API version is deprecated for the following Strimzi resources:

* `Kafka`
* `KafkaConnect`
* `KafkaConnectS2I`
* `KafkaMirrorMaker`
* `KafkaTopic`
* `KafkaUser`

> These resources that use the API version `{KafkaApiVersionPrev}` must be updated to use `{KafkaApiVersion}`.

Reviewer: ```suggestion
Update these resources from `{KafkaApiVersionPrev}` to use the `{KafkaApiVersion}` API version.
```

    upgrade: "Upgraded to {KafkaApiVersion}"
----

> The following sections contain a resource specific conversion guides:

Reviewer: ```suggestion
The following procedures contain the steps to update resources to use the `{KafkaApiVersion}` API version:
```
